Fastfile

```diff
@@ -140,10 +134,33 @@ def _build_with_gym()
   end
 end
 
+def _get_export_method()
+  ENV['GYM_EXPORT_OPTIONS'].nil? ? _fallback_to_enterprise_export_method() : get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: "method")
+end
+
+def _fallback_to_enterprise_export_method()
+  UI.message("Falling back to enterprise `export_method` since `GYM_EXPORT_OPTIONS` is not defined")
+  "enterprise"
+end
+
 def _update_team_id_if_necessary()
-  if !ENV['FL_PROJECT_SIGNING_PROJECT_PATH'].to_s.strip.empty? && !ENV['FL_PROJECT_TEAM_ID'].to_s.strip.empty?
-    update_project_team
+  project_path = ENV['FL_PROJECT_SIGNING_PROJECT_PATH']
+  team_id = _get_team_id()
+  if !project_path.to_s.strip.empty? && !team_id.to_s.strip.empty?
```

Comment:
I think the double negation here makes the code a bit unreadable.
Is it possible to go for something like:
`project_path.to_s.strip.present? && team_id.to_s.strip.present?`

or we do:
`project_path.to_s.strip.empty? || team_id.to_s.strip.empty?`
and put the "success" flow in the after the `else`

Reply:
Prefer the former. Nice! Will update

Reply:
@NikolaKirev updated